src/base.config.ts

```diff
@@ -93,7 +93,8 @@ export default function webpackConfigFactory(args: any): WebpackConfiguration {
 			filename: '[name].js',
 			jsonpFunction: getJsonpFunctionName(packageName),
 			libraryTarget: 'umd',
-			path: path.resolve('./output')
+			path: path.resolve('./output'),
+			publicPath: '/'
```

Comment:
doesn't this now make this non portable for anything other than a root?

Reply:
That's a fair point. I'll remove this in favor using just `'/'` directly in the middleware options, since I don't know that we can safely specify a `publicPath` for any application.